ha/k8s_setup/ha_setup.py

```diff
@@ -252,10 +252,10 @@ def process(self):
             # So, to get the cluster_id field from Confstore, we need machine_id
             self._cluster_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}cluster_id')
             # site_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}site_id')
-            self._site_id = '1'
+            self._site_id = NOT_DEFINED
             # rack_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}rack_id')
-            self._rack_id = '1'
-            self._storageset_id = '1'
+            self._rack_id = NOT_DEFINED
+            self._storageset_id = NOT_DEFINED
             conf_file_dict.update({'COMMON_CONFIG': {'cluster_id': self._cluster_id, 'rack_id': self._rack_id, 'site_id': self._site_id}})
```

Comment:
Can we push this storageset_id as well to the HA conf file? @ArchanaLimaye ?

Reply:
Yes we can do that

Reply:
Ok, lets take it separately.